README.md

```diff
@@ -315,6 +319,59 @@ To update your Global Response Set, add one or more rows to the spreadsheet. To
 
 Caveat: deleting a response, and then re-adding the same response later will result in iAuditor Analytics dashboard treating these as different responses. This is because the new response will have a different internal identifier than the deleted response had. To update a response while keeping the same internal identifier you will need to use the response set API directly, instead of this tool. See the iAuditor developer portal for more details.
 
+### The Export Users tool
+
+This tool exports a list of users and their groups from iAuditor into a CSV file named `iauditor_users.csv`.
+
+To run it:
+Open a command-line terminal and navigate to the directory called `safetyculture-sdk-python/tools/export_user`.
+Run the following command:
+```
+python export_user.py --token <YOUR_IAUDITOR_API_TOKEN>
+```
+The exported CSV file will be saved in the current working directory.
+If the file already exists, it is overwritten.
+The exported CSV file columns contain the following user information:
+- email
+- lastname
+- firstname
+- groups
+
+The field `groups` contains a comma-separated list of all iAuditor groups the user is a member of. All fields are string values.
+Note: comma-separated group names are not supported. If a group name contains a comma, it will be listed as is.
+
+### The Import Users tool
+
+This tool imports a list of users from a CSV file and assigns them to the specified groups in iAuditor in the organisation of the user whose API token is used.
+The input file specified in the command line arguments has the following structure:
+
+| email| lastname| firstname| groups|
+|---|---|---|---|
+|johnsmith@example.com |Smith|John|Group 1, Group 2|
+|johndoe@example.com|Doe|John|Group 3|
+|jasonR@example.com|R|Jason| |
+
+See `example_user.csv` in `safetyculture-sdk-python/tools/import_user` for an example.
+
+The first row must contain the column headings and will be ignored by the tool.
+
+The groups specified must have been created in iAuditor before running the tool. If a group doesn't exist, the user will not be added to that group.
+
+To run the tool:
+Open a command-line terminal and navigate to the directory called `safetyculture-sdk-python/tools/import_user`
+Run the following command:
+```
+python import_users.py --token <YOUR_IAUDITOR_API_TOKEN> --file <FULL_PATH_TO_CSV_FILE>
```

Comment:
Does it have to be CSV file? Can it load an XLSX file too?

Reply:
I think for xls  we require pythons xlsxwriter library..so this only supports csv.